pkg/utils/utils.go

```diff
@@ -16,3 +16,23 @@ func ParseSymbol(address uint64, table *helpers.KernelSymbolTable) *helpers.Kern
 	hookingFunction.Owner = strings.TrimSuffix(hookingFunction.Owner, "]")
 	return hookingFunction
 }
+
+func MatchFilter(filters []string, argValStr string) bool {
```

Comment:
As said, I think this should be either a method of `StringFilter` or receive the struct as an argument, in order to enforce the struct as a contract.
Also needs tests.